aten/src/ATen/native/SummaryOps.cpp

```diff
@@ -65,4 +134,112 @@ _bincount_cpu(const Tensor& self, const Tensor& weights, int64_t minlength) {
   });
 }
 
+
+std::tuple<Tensor,Tensor> _histogram_cpu(
+    const Tensor& self,
+    int64_t bins,
```

Comment:
"bins"->"nbins" to be consistent with above

Reply:
Done.